src/allencell_ml_segmenter/prediction/service.py

```diff
@@ -39,13 +33,6 @@ def __init__(self, model: PredictionModel):
     def handle_event(self, event: Event) -> None:
         pass
 
-    def extract_preprocessing_method(self) -> None:
```

Comment:
Was this dead code, left over from a UI we r emoved?

Reply:
this is dead code